.github/workflows/build.yml

```diff
@@ -17,6 +17,19 @@ jobs:
         with:
           java-version: '14'
           distribution: 'adopt'
+ 
+      - name: Checkout OpenSearch
```

Comment:
Hm ... why we are building `opensearch-project/opensearch` instead of using published snapshots from `https://aws.oss.sonatype.org/content/repositories/snapshots/`?

Reply:
Ah my bad! I can use that for build and unit test. I will update the workflows for the same.